deeplearning4j-keras/src/main/java/org/deeplearning4j/keras/Server.java

```diff
@@ -0,0 +1,20 @@
+package org.deeplearning4j.keras;
+
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import py4j.GatewayServer;
+
+/**
+ * Main class for the DL4J-as-Keras-backend. Simply launches py4j GatewayServer with
+ * an entry point exposing API available on Python side.
+ */
+@Slf4j
```

Comment:
I always kind of thought that adding author just makes the file longer as you can check the real author just by looking on blame of the file ;) but ok - I'll add this :)